tests/Api/Admin/ProductVariantTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Tests\Api\Admin;
+
+use Sylius\Tests\Api\JsonApiTestCase;
+use Symfony\Component\HttpFoundation\Response;
+
+final class ProductVariantTest extends JsonApiTestCase
+{
+    /** @test */
+    public function it_denies_access_to_a_products_list_for_not_authenticated_user(): void
+    {
+        $this->loadFixturesFromFile('product_variant.yaml');
+
+        $this->client->request('GET', '/api/v2/admin/product-variants/MUG');
+
+        $response = $this->client->getResponse();
+        $this->assertSame(401, $response->getStatusCode());
```

Comment:
```suggestion
        $this->assertSame(Response::HTTP_UNAUTHORIZED, $response->getStatusCode());
```
wouldnt it be easier to read?